src/interfaces/uproxy_core_api.ts

```diff
@@ -290,5 +302,12 @@ export interface CoreApi {
   // This may also invoke an OAuth flow, in order to perform operations
   // with the cloud computing provider on the user's behalf.
   cloudInstall(args:CloudInstallArgs): Promise<void>;
+
+  // Invokes OAuth flow and destroys uProxy cloud server on the 
+  // provider specified.
+  cloudDestroy(providerName :string) : Promise<void>;
```

Comment:
I ended up changing it to CloudOperationArgs (but could always change it back). I think that at least right now it's not too complicated, and it might not be too long until we always have to pass providerName and serverName if we let people name their servers.